Makefile

```diff
@@ -148,6 +148,14 @@ test-snuba: create-db
 		-vv --cov . --cov-report="xml:.artifacts/snuba.coverage.xml"
 	@echo ""
 
+test-snuba-ci: create-db
+	@echo "--> Running snuba tests"
+	pytest tests \
+		-k snuba_ci \
```

Comment:
use `-m` here. this will search for any test that contains the string `snuba_ci`

Reply:
That picks up every test with the name `snuba` anywhere in it, is seems way too many

Reply:
no, it's the other way around. `-k` does what you describe.

Reply:
oops, updated